frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -1262,6 +1263,30 @@ export const dashboardLogic = kea<dashboardLogicType>([
                 actions.loadDashboard({ action: 'update' })
             }
         },
+        [dashboardsModel.actionTypes.updateDashboardSuccess]: ({ dashboard }) => {
+            if (values.dashboard == null || dashboard == null) {
+                return
+            }
+            if (values.dashboard.id !== dashboard.id) {
+                return
+            }
+            const newDashboard = { ...dashboard }
+            // Check if there are temporary filters
+            const temporaryFilters = values.temporaryFilters
+            if (
+                temporaryFilters.properties != null ||
+                temporaryFilters.breakdown_filter != null ||
+                temporaryFilters.date_from != null ||
+                temporaryFilters.date_to != null
+            ) {
+                // If filter value is not applied and preview, do not replace
+                if (!values.filtersUpdated) {
```

Comment:
There is an edge case here, which is that if you have some filters that are previewed and some filters that are not, this will clear all the previews. 

Steps to reproduce:
1) Set the date filter to today, and apply
2) Set a filter but do not apply
3) Update the name, notice that the charts revert back to the unfiltered state.

https://github.com/user-attachments/assets/ac782923-c843-4887-88b3-83ec891a4c0f

Reply:
I suspect maybe the url approach can fix this; or we need to update `filtersUpdated` to track the individual updated state instead of just returning a boolean. Let me know if I'm not making sense!